doc/source/user_guide/reshaping.rst

```diff
@@ -312,6 +313,27 @@ user-friendly.
   dft
   pd.wide_to_long(dft, ["A", "B"], i="id", j="year")
 
+
+Keeping the index around
+~~~~~~~~~~~~~~~~~~~~~~~~
+When transforming a DataFrame using :func:`~pandas.melt`, original index values can be kept around by setting the ``keep_index`` parameter to True. This will however duplicate the index values.
```

Comment:
```suggestion

When transforming a DataFrame using :func:`~pandas.melt`, original the index values can be kept around by setting the ``keep_index`` parameter to True. This will however duplicate the index values.
```